clinica/resources/templates/pipeline_template/cli.py.j2

```diff
@@ -26,24 +26,37 @@ class {{ pipeline.class_name }}CLI(ce.CmdParser):
     def define_options(self):
         """Define the sub-command arguments
         """
+        from clinica.engine.cmdparser import PIPELINE_CATEGORIES
+
+        # Clinica compulsory arguments (e.g. BIDS, CAPS, group_id)
+        clinica_comp = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_COMPULSORY'])
+        clinica_comp.add_argument("bids_directory",
+                                  help='Path to the BIDS directory.')
+        clinica_comp.add_argument("caps_directory",
+                                  help='Path to the CAPS directory.')
+        clinica_comp.add_argument("group_id",
```

Comment:
I would comment lines for `group_id`. For a new developer, he/she will be likely to work with BIDS datasets as inputs and CAPS as outputs.

What do you think?

Reply:
Agree